bones/booleanBone.py

```diff
@@ -23,6 +23,14 @@ def singleValueFromClient(self, value, skel, name, origData):
 		else:
 			return False, None
 
+	def getEmptyValue(self):
+		return False
+
+	def isEmpty(self, rawVale: Any):
+		if rawVale is self.getEmptyValue():
+			return True
+		return bool(rawVale)
```

Comment:
Is `return bool(rawValue)` correct?
If _rawValue_ is _True_ it would return `True` which means the field is empty.

Reply:
Correct. Is Fixed.